package.json

```diff
@@ -3,18 +3,20 @@
   "version": "0.4.7",
   "description": "A star rater in react.js",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "directories": {
     "example": "example"
   },
   "scripts": {
     "test": "jest --verbose",
     "compile-scss": "node-sass src/react-rater.scss",
-    "build-lib": "rm -rf lib && mkdir lib && babel src --out-dir lib; npm run -s compile-scss >> lib/react-rater.css",
+	"build-lib-windows": "(if exist lib rd /s /q lib) && mkdir lib && babel src --out-dir lib && copy src\\index.d.ts lib && npm run -s compile-scss >> lib/react-rater.css && copy src\\react-rater-sass.scss lib ",
+    "build-lib": "rm -rf lib && mkdir lib && babel src --out-dir lib && cp src/index.d.ts lib/index.d.ts; npm run -s compile-scss >> lib/react-rater.css; cp src/react-rater-sass.scss lib/",
```

Comment:
Thank you for adding the script!
Would you mind replace the tab indent with space so that it's consistent?
Thank you 👍